esx_service/cli/vmdkops_admin.py

```diff
@@ -845,7 +859,24 @@ def tenant_create(args):
     if error_info:
         return operation_fail(error_info)
     else:
-        print "tenant create succeeded"    
+        print "tenant create succeeded"
+
+def tenant_rename(args):
+    """ Handle tenant create command """
+    error_info, tenant = get_tenant_from_db(args.name)
+    if error_info:
+        return operation_fail(error_info)
+    
+    if not tenant:
+        error_info = "Tenant {0} does not exist".format(args.name)
+        return operation_fail(error_info)
+
+    error_info = tenant.set_name(_auth_mgr.conn, args.new_name)
```

Comment:
`tenant.set_name` only updates DB record. TenantName is also used as folder name under dockvols. Once tenant name is changed, list/attach/detach/get will stop working for existing volumes. Right?